portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -25,18 +25,24 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import java.time.format.DateTimeFormatter;
+import java.time.*; 
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/* Servlet that creates a comment*/
 @WebServlet("/new-comment")
 public class NewCommentServlet extends HttpServlet {
   
+  private static final DateTimeFormatter commentDateFormat = DateTimeFormatter.ofPattern("yyyy/MM/dd");
+  
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("commentText");
     long timestamp_millis = System.currentTimeMillis();
-
+    String currentDate = commentDateFormat.format(LocalDateTime.now());
+    
     Entity commentEntity = new Entity("Comment");
     commentEntity.setProperty("content", text);
+    commentEntity.setProperty("currentDate", currentDate);
```

Comment:
I don't have a preference as to whether properties are camelCase or snake_case, but they should probably be consistent (for retaining your own sanity in the future).

Reply:
You didn't actually fix this